File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        int once = 0;
        int twice = 0;

        for(int num : nums) {
            if(num <=9) once += num;
            else twice += num;
        }
        return (once > twice) || twice > once;
    }
};

File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.py
class Solution(object):
    def canAliceWin(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        once = 0
        twice = 0
        for num in nums:
            if (num <= 9) : once += num
            else : twice += num
        return once > twice or twice > once
        
        